enterprise/server/remote_execution/containers/ociruntime/BUILD

```diff
@@ -39,6 +40,13 @@ go_library(
     ],
 )
 
+genrule(
```

Comment:
`copy_file` from skylib with `allow_symlink = True` is much more remote friendly - it doesn't even run an execution and also forwards the digest of the input.

Reply:
Ah, nice - we have a bunch of places that do this; might clean these up in a later PR.